Platform/QemuBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -340,6 +340,17 @@ UpdateOsBootMediumInfo (
   OsBootOptionList->OsBootOptionCount++;
   BootOption++;
 
+  // Boot file image from USB
```

Comment:
I think it would be better to decouple the the USB keyboard logic from USB image boot structure. Even if they are the same USB controller, I think users may find this confusing.

Reply:
This requires a generic mechanism to provide device map information.   An initial thought, we could provide a system device mapping table which can be consumed by many drivers.

Reply:
@mauricema I think that's a good idea! Perhaps something akin to Device Trees.